src/styles/styles.js

```diff
@@ -361,6 +361,18 @@ const styles = {
         backgroundColor: themeColors.offline,
     },
 
+    floatingActionButton: {
+        backgroundColor: colors.green,
+        position: 'absolute',
+        height: 40,
```

Comment:
There should be a variable for this value - I believe it is `variables.componentSizeNormal`

Reply:
Done!